scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
CROSS_COMPILE is zlib thing, and zlib's configure is not produced by autotools - it is hand-written.

As to configure - yes, but they should be looking for `${ac_tool_prefix}gcc` and `${ac_tool_prefix}cc`, and `${ac_tool_prefix}` is set to `${host_alias}-` if host alias is not empty, and it shouldn't be empty in our case since we pass `--host`. So the question stands, why didn't GMP's configure find this `x86_64-build_unknown-freebsd11.0-gcc` wrapper?